main-client.js

```diff
@@ -26,7 +27,20 @@ export const createMeteorNetworkInterface = (givenConfig) => {
   if (config.useMeteorAccounts) {
     networkInterface.use([{
       applyMiddleware(request, next) {
-        const currentUserToken = Accounts._storedLoginToken();
+        
+        // cookie login token created by meteorhacks:fast-render and caught during server-side rendering by rr:react-router-ssr
+        const { loginToken: cookieLoginToken } = config;
+        // Meteor accounts-base login token stored in local storage, only exists client-side
+        const localStorageLoginToken = Meteor.isClient && Accounts._storedLoginToken();
+        
+        // on initial load, prefer to use the token grabbed server-side if existing
+        let currentUserToken = cookieLoginToken || localStorageLoginToken;
+
+        // ...a login token has been passed to the config, however the "true" one is different ⚠️
+        if (Meteor.isClient && cookieLoginToken && cookieLoginToken !== localStorageLoginToken) {
```

Comment:
I've had hard time debugging that, but I believe this happened after a refresh of the browser or hot-reloading of Meteor. The cookie injected by fast-render via `reactrouter:react-router-ssr` was "still" kept in store even if the local storage token has became `undefined`.

It may due to our way of hijacking `rr:ssr`: https://github.com/TelescopeJS/Telescope/blob/devel/packages/_react-router-ssr/lib/client.jsx#L58

...As I was writing and thinking more and more about your comment, I realized I didn't need at all to do this, passing the cookie from the client!! I've been caught by the idea of `react-cookie`'s isomorphic cookie! And messed the thing myself haha! 😄 

*Conclusion:* shall we remove this extra security?